components/scream/scripts/utils.py

```diff
@@ -474,9 +474,10 @@ def ensure_pip():
 
         ensurepip.bootstrap(user=True)
 
-        site.main() # needed to "rehash" available libs
+        # needed to "rehash" available libs
+        site.main() # pylint: disable=no-member
 
-        import pip
+        import pip # pylint: disable=import-error
```

Comment:
I know nothing about pylint; is this comment string parsed by pylint?

Reply:
Yes, it suppresses a pylint error. Pylint can have false positives.